instrumentation/aws-sdk/aws-sdk-1.11/library/src/main/java/io/opentelemetry/instrumentation/awssdk/v1_11/RequestAccess.java

```diff
@@ -90,6 +150,12 @@ private RequestAccess(Class<?> clz) {
     getTableName = findAccessorOrNull(clz, "getTableName");
     getTopicArn = findAccessorOrNull(clz, "getTopicArn");
     getTargetArn = findAccessorOrNull(clz, "getTargetArn");
+    getStateMachineArn = findAccessorOrNull(clz, "getStateMachineArn");
+    getStepFunctionsActivityArn = findAccessorOrNull(clz, "getActivityArn");
+    getSnsTopicArn = findAccessorOrNull(clz, "getTopicArn");
```

Comment:
there is already `getTopicArn` that is used to fill `messaging.destination.name` attribute for sns messages